packages/lesswrong/components/async/localStorageHandlers.js

```diff
@@ -0,0 +1,66 @@
+import ls from 'local-storage';
+
+export const commentLSHandlers = {
+  get: ({doc, name}) => {
+    if (doc && doc._id) { // When restoring the edit state for a specific comment, ask for permission
```

Comment:
If I'm understanding this branch correctly, the options are:
 * You're restoring a specific comment by ID
 * You're restoring "the comment you were working on as a reply to `postId`"
 * You're restoring "the comment you were working on as a reply to `parantCommentId`"
This could use a clarifying comment (I was confused what was going on on the first pass).